airbyte-integrations/connectors/source-shopify/integration_tests/configured_catalog.json

```diff
@@ -84,6 +84,18 @@
       "cursor_field": ["updated_at"],
       "destination_sync_mode": "append"
     },
+    {
+      "stream": {
+        "name": "orders_graph_ql",
+        "json_schema": {},
+        "supported_sync_modes": ["incremental", "full_refresh"],
+        "source_defined_cursor": true,
+        "default_cursor_field": ["createdAt"]
```

Comment:
I think, as well, we have to implement updates of created orders, using updatedAt fields

Reply:
You can choose on Airbyte Full refresh